src/lexer.c

```diff
@@ -2403,21 +2419,24 @@ void Lexer::getDocComment(Token *t, unsigned lineComment)
 
     // Combine with previous doc comment, if any
     if (*dc)
-        *dc = combineComments(*dc, (utf8_t *)buf.data);
+        *dc = combineComments(*dc, (utf8_t *)buf.data, newParagraph);
     else
         *dc = (utf8_t *)buf.extractData();
 }
 
 /********************************************
  * Combine two document comments into one,
- * separated by a newline.
+ * separated by an extra newline if newParagraph != 0.
  */
 
-const utf8_t *Lexer::combineComments(const utf8_t *c1, const utf8_t *c2)
+const utf8_t *Lexer::combineComments(const utf8_t *c1, const utf8_t *c2, int newParagraph)
 {
-    //printf("Lexer::combineComments('%s', '%s')\n", c1, c2);
+    //printf("Lexer::combineComments('%s', '%s', '%i')\n", c1, c2, newParagraph);
 
     const utf8_t *c = c2;
+    
```

Comment:
Oh, it didn't give a line number so I thought it meant "at the end of the entire file". Never seen indentation in a blank line be an issue before. None of my editors eliminate or highlight that sort of thing by default. Will fix.